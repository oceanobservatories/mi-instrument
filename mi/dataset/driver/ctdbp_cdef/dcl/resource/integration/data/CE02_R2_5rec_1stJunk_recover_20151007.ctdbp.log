2015/10/07 18:41:16.281 [ctdbp:DLOGP3]:Logger started: Idle state, without initialize
2015/10/07 18:45:06.115 [ctdbp:DLOGP3]:Instrument Started [Power On] 2015/10/07 18:45:19.082 # 13.3687,  0.00002,    0.051, 07 Oct 2015 18:45:18
2015/10/07 18:45:29.076 # 13.3701,  0.00002,    0.053, 07 Oct 2015 18:45:28
2015/10/07 18:45:39.076 # 13.3652,  0.00002,    0.052, 07 Oct 2015 18:45:38
2015/10/07 18:45:49.089 # 13.3637,  0.00002,    0.051, 07 Oct 2015 18:45:48
2015/10/07 18:45:59.076 # 13.3652,  0.00002,    0.052, 07 Oct 2015 18:45:58
2015/10/07 18:46:09.076 # 13.3663,  0.00003,    0.051, 07 Oct 2015 18:46:08
2015/10/07 18:48:24.380 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
