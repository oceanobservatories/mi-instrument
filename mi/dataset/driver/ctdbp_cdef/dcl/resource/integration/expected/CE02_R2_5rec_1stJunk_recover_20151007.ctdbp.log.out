
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653164800 | recovered | 3653232329.0760002136 |          2 |         3653232328 | 3653232329.0760002136 |      port_timestamp |           ok |        2e-05 |             null |                     null |    0.053 | 13.3701
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653164800 | recovered | 3653232339.0760002136 |          2 |         3653232338 | 3653232339.0760002136 |      port_timestamp |           ok |        2e-05 |             null |                     null |    0.052 | 13.3652
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653164800 | recovered | 3653232349.0889997482 |          2 |         3653232348 | 3653232349.0889997482 |      port_timestamp |           ok |        2e-05 |             null |                     null |    0.051 | 13.3637
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653164800 | recovered | 3653232359.0760002136 |          2 |         3653232358 | 3653232359.0760002136 |      port_timestamp |           ok |        2e-05 |             null |                     null |    0.052 | 13.3652
 CE02SHSM | RID27 | 03-CTDBPC000 | 3653164800 | recovered | 3653232369.0760002136 |          2 |         3653232368 | 3653232369.0760002136 |      port_timestamp |           ok |        3e-05 |             null |                     null |    0.051 | 13.3663

(5 rows)
